tests/unittests/general/bulkparse_test.py

```diff
@@ -84,7 +84,7 @@ def test_three_lines_with_two_rows_should_be_counted_as_three(self):
             b"room1:10.0.0.187:myorg:OTHER:SNMP v2c read profile::\n"
         )
         b = bulkparse.NetboxBulkParser(data)
-        out_data = list(b)
+        list(b)
```

Comment:
Makes no sense. If you run ruff again this line will probably be deleted.

Reply:
No, actually not, if you try deleting it the test will fail

Reply:
IIRC, there is some generator magic with the bulk parser classes.  The generator needs to be fully consumed for all the input text to be fully parsed and the line count be updated.

The `out_data` was just a throwaway variable here, but perhaps a better way to signal the intent to consume the entire generator/iterator, without otherwise caring about each returned value, is to instead do something akin to:
```suggestion
        itertools.consume(b)
```